<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1575</width>
    <height>987</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0">
    <item row="4" column="0">
     <layout class="QGridLayout" name="bottomLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="dislikeBtn">
        <property name="text">
         <string>Dislike</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="likeBtn">
        <property name="text">
         <string>Like</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="previousBtn">
        <property name="text">
         <string>Previous</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="nextBtn">
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="prevPostBtn">
        <property name="text">
         <string>Previous Post</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="nextPostBtn">
        <property name="text">
         <string>Next Post</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QPlainTextEdit" name="infoText">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>Post Information</string>
        </property>
        <property name="backgroundVisible">
         <bool>false</bool>
        </property>
        <property name="centerOnScroll">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="topLayout" rowstretch="0,0">
      <item row="0" column="3">
       <widget class="QLineEdit" name="subredditEntry">
        <property name="text">
         <string>wallpaperdump</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Post ID</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="postIDEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="inputMask">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="previewBtn">
        <property name="text">
         <string>Preview</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QGroupBox" name="filterGroup">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>Filter</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="nsfwCheck">
           <property name="text">
            <string>NSFW</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="sfwCheck">
           <property name="text">
            <string>SFW only</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="skipCheck">
           <property name="text">
            <string>Skip rated</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="nonImageCheck">
           <property name="text">
            <string>Include non-images</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="keywordEntry">
           <property name="placeholderText">
            <string>Title contains</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Subreddit</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Step Time:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="stepSpinner">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="idLabel">
          <property name="text">
           <string>Image ID: -1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="centerLayout">
      <item row="0" column="0">
       <widget class="QWebView" name="webView">
        <property name="url">
         <url>
          <string>about:blank</string>
         </url>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1675</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="saveAction"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="saveAction">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="setLastAction">
   <property name="text">
    <string>Set Last ID</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
